src/Symfony/Component/DependencyInjection/Compiler/ResolveInstanceofConditionalsPass.php

```diff
@@ -77,11 +77,13 @@ private function processDefinition(ContainerBuilder $container, $id, Definition
             foreach ($instanceofDefs as $key => $instanceofDef) {
                 /** @var ChildDefinition $instanceofDef */
                 $instanceofDef = clone $instanceofDef;
-                $instanceofDef->setAbstract(true)->setParent($parent ?: 'abstract.instanceof.'.$id);
-                $parent = 'instanceof.'.$interface.'.'.$key.'.'.$id;
+                $instanceofDef->setAbstract(true)->setParent($parent ?: "abstract.instanceof.$id");
```

Comment:
please revert these changes, they are unrelated (did fabbot suggest them? if yes, please it :) )

Reply:
Not fabbot, just did it because string interpolation run faster in PHP7, but I can revert it.

Reply:
yes please do, that's unrelated